docs/superpowers/specs/2026-06-09-rob-347-us-budget-fx-basis-design.md

```
@@ -0,0 +1,135 @@
+  null이면 basis 선택을 따른다.
+
+### 3.2 budget 신호 소스 (propose 내부)
+propose가 portfolio snapshot payload에서 추출:
+- `usd_buying_power = (buying_power or {}).get("usd")` (0/None 가능)
+- `krw_orderable_reference = (buying_power or {}).get("krw")` (**reference only**)
+- `operator_override = operator_budget_override_usd` (request 인자)
+이를 `budget_state` dict로 정규화(아래 헬퍼 입력).
+
+### 3.3 budget-aware demotion (verdict, B의 다음 단계)
+`action_verdict.py` 에 **순수 헬퍼** 추가(서명 미변경 원칙 동일):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)